roles/confluent.variables/vars/main.yml

```diff
@@ -496,6 +496,7 @@ kafka_connect_properties:
       bootstrap.servers: "{{ groups['kafka_broker'] | default(['localhost']) | join(':' + kafka_broker_listeners[kafka_connect_kafka_listener_name]['port']|string + ',') }}:{{kafka_broker_listeners[kafka_connect_kafka_listener_name]['port']}}"
       producer.bootstrap.servers: "{{ groups['kafka_broker'] | default(['localhost']) | join(':' + kafka_broker_listeners[kafka_connect_kafka_listener_name]['port']|string + ',') }}:{{kafka_broker_listeners[kafka_connect_kafka_listener_name]['port']}}"
       consumer.bootstrap.servers: "{{ groups['kafka_broker'] | default(['localhost']) | join(':' + kafka_broker_listeners[kafka_connect_kafka_listener_name]['port']|string + ',') }}:{{kafka_broker_listeners[kafka_connect_kafka_listener_name]['port']}}"
+      confluent.license.topic: _confluent-license
```

Comment:
I think default topic for connectors license is `_confluent-command`
https://docs.confluent.io/kafka-connect-replicator/current/configuration_options.html#cp-license

Reply:
@jeqo Great catch! Connect differs from both Schema registry and KSQL which use `_confluent-license`, I vaguely recall getting confused on this previously.